crates/file-explorer/src/fs.rs

```diff
@@ -28,6 +29,7 @@ pub struct File {
     pub metadata: Metadata,
 }
 
+#[allow(dead_code)]
```

Comment:
Please remove

```suggestion
```

Reply:
I don't know why there is dead code in the crate file-explorer but the warnings are present.